groups/rds/rds.tf

```diff
@@ -0,0 +1,130 @@
+# ------------------------------------------------------------------------------
+# RDS Security Group and rules
+# ------------------------------------------------------------------------------
+module "rds_security_group" {
+
+  source  = "terraform-aws-modules/security-group/aws"
+  version = "~> 3.0"
+
+  name        = "sgr-smartview-rds-001"
+  description = "Security group for the smartview RDS database"
+  vpc_id      = data.aws_vpc.vpc.id
+
+  ingress_cidr_blocks = concat(local.admin_cidrs, var.rds_onpremise_access)
+  ingress_rules       = ["oracle-db-tcp"]
+  ingress_with_cidr_blocks = [
+    {
+      from_port   = 5500
+      to_port     = 5500
+      protocol    = "tcp"
+      description = "Oracle Enterprise Manager"
+      cidr_blocks = join(",", concat(local.admin_cidrs, var.rds_onpremise_access))
+    }
+  ]
+  ingress_with_source_security_group_id = []
+
+  egress_rules = ["all-all"]
+}
+
+# ------------------------------------------------------------------------------
+# RDS Instance
+# ------------------------------------------------------------------------------
+module "smartview_rds" {
+  source  = "terraform-aws-modules/rds/aws"
+  version = "2.23.0" # Pinned version to ensure updates are a choice, can be upgraded if new features are available and required.
```

Comment:
Could be changed to a variable and moved to the profile vars. That would allow versions to be pinned per environment - so the future upgrades mentioned in the comments could be tested in one env (staging, etc.) before moving to Live.

Reply:
Terraform will not accept a variable in this position, so it'll need to remain as-is